packages/common/src/store/user-list/notifications/types.ts

```diff
@@ -3,7 +3,7 @@ import { UserListStoreState } from '~/store/user-list/types'
 export const NOTIFICATIONS_USER_LIST_TAG = 'NOTIFICATION'
 
 export type NotificationUsersPageOwnState = {
-  id: string | null
+  notification: any | null
```

Comment:
is there no notofication type we can use here?

Reply:
yeah will look